types/react-navigation/index.d.ts

```diff
@@ -569,8 +569,8 @@ export function TabNavigator(
 export interface StackNavigatorScreenOptions {
   title?: string;
   headerVisible?: boolean;
-  headerTitle?: string|React.ReactElement<any>;
-  headerBackTitle?: string|null;
+  headerTitle?: string | React.ReactElement<any>;
+  headerBackTitle?: string | null;
```

Comment:
sorry, It's my fault.
I haven't said it clearly enough.

I mean remove the null in optional value in this line.

because the optional value could be null, and there is no need to declare null twice.

`headerBackTitle?: string | null;`
=>
`headerBackTitle?: string;`

your linter is better,  and no need to reset the format.

😳